### ISSUE TITLE
Integration of Dynamic RDT definitions

### ISSUE THREAD

**Issue by shivenbhatt1** [onaio/rdt-standard#460]
Blocked by closing https://github.com/onaio/Covid19/issues/15

Set up a meeting with Sam and PLD on dynamic tasking

**Comment by vincent-karuri** [comment]
Duplicate of https://github.com/onaio/rdt-standard/issues/511
